### ISSUE TITLE
How to create a specific style for {{ }}

### ISSUE THREAD

**Issue by danielCarlosCE** [noties/Markwon#140]
* **Markwon version**: 3.0.1

Hey there! First, thanks for the lib. It's awesome. 

I have this app where I also need to show **underlined words**. I'm using this pattern`{{}}` on iOS. I'm not sure if I didn't do a good job searching, but I couldn't figure out from the documentation if I actually can do that. 

Thanks

**Comment by noties** [comment]
Hello @danielCarlosCE !

Well, if you just need to <ins>underline</ins> words, `HtmlPlugin` can be used. Your text should be enclosed in `<u></u>` or `<ins></ins>` tags to be underlined. There is also a [commonmark-java](https://github.com/atlassian/commonmark-java) extension (`ext-ins`) that allows underlining with `++` delimiter characters:

```java
// gradle: implementation 'com.atlassian.commonmark:commonmark-ext-ins:0.12.1'

final Markwon markwon = Markwon.builder(context)
        .usePlugin(new AbstractMarkwonPlugin() {
            @Override
            public void configureParser(@NonNull Parser.Builder builder) {
                // register extension
                builder.extensions(Collections.singletonList(InsExtension.create()));
            }

            @Override
            public void configureSpansFactory(@NonNull MarkwonSpansFactory.Builder builder) {
                // register span-factory
                builder.setFactory(Ins.class, new SpanFactory() {
                    @Override
                    public Object getSpans(@NonNull MarkwonConfiguration configuration, @NonNull RenderProps props) {
                        return new UnderlineSpan();
                    }
                });
            }
        })
        .build();
```

But if you wish to be consistent between platforms and use `{{}}` you will have to create own delimiter processor, it's pretty easy (you can look at `ext-ins` for a reference). Also, I'm just publishing a snapshot for the next `4.0.0` version that has `SimpleExtPlugin` that allows doing this:

```java
final Markwon markwon = Markwon.builder(this)
        .usePlugin(SimpleExtPlugin.create(plugin -> plugin
                .addExtension(2, '{', '}', (configuration, props) -> new UnderlineSpan())))
        .build();
```

**Comment by noties** [comment]
`SimpleExtPlugin` is published with the latest `4.0.0` release.
